src/haddock/clis/cli_analyse.py

```diff
@@ -301,8 +301,8 @@ def analyse_step(step, run_dir, capri_dict, target_path, top_cluster, format, sc
     else:
         log.info(f"step {step} is caprieval, files should be already available")
         ss_fname = Path(run_dir, f"{step}/capri_ss.tsv")
-        clt_fname = Path(run_dir, f"{step}/capri_clt.tsv")
         shutil.copy(ss_fname, target_path)
+        clt_fname = Path(run_dir, f"{step}/capri_clt.tsv")
```

Comment:
what is this change? probably related to some space..try to avoid introducing such changes, as they make the history messy